2.0.62-RELEASE Feature#224

Elisenda-LV opened the pull request:
Feature#224 actualizada con 2.0.60-RELEASE
Pendiente revisar configuración Jest para poder pasar testing de crypto-browserify.

Elisenda-LV committed:
install crypto-browserify

Elisenda-LV committed:
add helpers: cookie-encryption

Elisenda-LV committed:
cookie-encryption service implemented

Elisenda-LV committed:
cookie-encryption testing implemented

Elisenda-LV committed:
Testing passed successfully

Elisenda-LV committed:
rename helpers and add HelpersModule to AppModule

Elisenda-LV committed:
Compodoc comment added

Elisenda-LV committed:
2.0.57-RELEASE

Elisenda-LV committed:
testing bug fixed: HttpClientTestingModule added

Elisenda-LV committed:
Merge branch 'develop' into feature#224

Elisenda-LV committed:
2.0.62-RELEASE

jonatanvicente committed:
Fixing error on tests & tuning names

jonatanvicente commented:
* Renombro CookieEncryptionService por CookieEncryptionHelper
* Elimino ItinerariesService del test, ahora pasan correctamente
* Pendiente resolución de problemas de importación de 'crypto-browserify' en CookieEncryptionHelper

jonatanvicente committed:
Renaming file

Tuning config of cookie-encryption.helper

Implement Cookie encryption with API Web Crypto instead crypto-js or crypto-browserify

Elisenda-LV committed:
Merge branch 'develop' into feature#224

Elisenda-LV committed:
Cookie encryption helper linked to auth.service

Elisenda-LV committed:
externalize password

Elisenda-LV committed:
Clean up code

Elisenda-LV commented:
- Limpio código de cookie-encryption.helper.ts y cookie-encryption.helper.spec.ts
- Externalizo password a environments.ts.
- Anclo helper en auth.service.
- Mantengo el testing deshabilitado.

jonatanvicente commented:
Es necesario habilitar testing, actualmente failed 
![Pasted image 1](https://github.com/IT-Academy-BCN/ita-challenges-frontend/assets/1706861/ffd7cf1f-fc1d-466b-b2fa-47226726d80f)
